modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -82,7 +96,7 @@
 |`./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-quay-dynamic`
 
 
-|RBAC  |`https://npmjs.com/package/@janus-idp/backstage-plugin-rbac/v/1.29.0[@janus-idp/backstage-plugin-rbac]` |1.29.0 
+|RBAC  |`https://npmjs.com/package/@janus-idp/backstage-plugin-rbac/v/1.29.4[@janus-idp/backstage-plugin-rbac]` |1.29.4 
```

Comment:
Wonder if it's worthwhile to make these attributes, and then only the attribute needs to be updated, and the latest version will be reflected everywhere the attribute is used in RHDH prod docs.

Reply:
Good point Gwynne, and this would normally be the best practice from a docs perspective...but for context, this table is actually automatically generated from a script (`rhdh-supported-plugins.sh`) created by Nick which extracts all the data including plugin versions from RHDH configuration files. So in this case, there would be no benefit it using version attributes.

Reply:
Ah, OK. Appreciate the context. I did not realize all the info for plugins listed in the table were generated by a script.

Out of curiosity, is this table the only place versions for plugins are listed? Though that might be a question more for a Doc Team sync...

OK. Appreciate the context. You can resolve the other attribute-related comments. Thanks!

Reply:
Yes it is @gwynnemonahan

Reply:
OK. Attributes don't make sense then ;) 

Thanks for the clarifications!